src/ConjugatePriors.jl

```diff
@@ -4,10 +4,19 @@ module ConjugatePriors
 
 using PDMats
 using Distributions
+using StatsFuns
 
 import Base: mean
 import Base.LinAlg: Cholesky
 
+import PDMats:
```

Comment:
this conflicts with master; you can fix by rebasing your PR on the latest master and resolving the conflict there.